### DISCUSSION TITLE
Not able to upgrade OS  - Computer reboots, but doesnt upgrade.

### DISCUSSION THREAD

**Discussion by kyevi** [grahampugh/erase-install#279] [Q&A]
*First of all. I would like to thank you for this awesome effort to help the community!*

Does anyone could help me on finfing out why Im getting this log? 
Im executing this: `/Library/Management/erase-install/erase-install.sh --os=12 --update --reinstall --current-user --depnotify --check-power --cleanup-after-use`
-----------------------------------------------------------------------------------------------------------------------------------
Result of command:
[erase-install] v26.2 script execution started: Fri Aug 19 09:35:58 -03 2022
   [erase-install] Caffeinating this script (pid=10632)
   [check_free_space] OK - 358 GB free/purgeable disk space detected
   [check_power_status] WARNING - No AC power detected
   [wait_for_power] Waiting for AC power...
   [wait_for_power] OK - AC power detected

   [erase-install] attempting to terminate the 'jamfHelper' process - Termination message indicates success
/Library/Management/erase-install/erase-install.sh: line 1078: 10662 Terminated: 15          "$jamfHelper" -windowType "utility" -title "${!dialog_power_title}" -description "${!dialog_power_desc} ${power_wait_timer_friendly}" -alignDescription "left" -icon "$dialog_confirmation_icon"

   [erase-install] Looking for existing installer app or pkg
   [find_existing_installer] Installer sparse image found at /Library/Management/erase-install/Install_macOS_12.5-21G72.sparseimage.
   [check_installer_is_valid] Checking validity of .
   [check_installer_is_valid] Using DTSDKBuild value from Info.plist
   [check_installer_is_valid] Installer Info.plist could not be found!
   [check_installer_is_valid] Build of existing installer could not be found!

   [erase-install] attempting to terminate the 'caffeinate' process - Termination message indicates success
/Library/Management/erase-install/erase-install.sh: line 1078: 10648 Terminated: 15          /usr/bin/caffeinate -dimsu -w $$
-----------------------------------------------------------------------------------------------------------------------------------

The computer reboots and its not updated.

**Comment by grahampugh** [comment]
I don't know why it's rebooting (have you set a Jamf policy to reboot by any chance?), but the installer in that machine looks like it is incomplete/corrupt. You'll need to delete it or run with --overwrite.
